### PR TITLE
Update release scripts

### PR DESCRIPTION
**Pull request by ebadyano**

### CODE REVIEW

File: changelog.py [hunk 1]
```diff
@@ -105,8 +105,8 @@ def main():
     print("### %s\n" % milestone_name)
 
     print_category("Highlights", prs(gh, milestone, with_labels="highlight"))
-    print_category("Enhancements", prs(gh, milestone, with_labels="enhancement", without_labels=[":Docs", "highlight"]))
-    print_category("Bug Fixes", prs(gh, milestone, with_labels="bug", without_labels=[":Docs", "highlight"]))
+    print_category("Enhancements", prs(gh, milestone, with_labels="enhancement", without_labels=[":Docs", "highlight", ":internal"]))
```
[the remark this answers is not in the document]
Comment (danielmitterdorfer) [reply changelog.py:108]: I think we should still include them but maybe in a separate category at the end?

[the remark this answers is not in the document]
Comment (DJRickyB) [reply changelog.py:108]: IDK.  If I look at https://github.com/elastic/rally/issues?q=label%3A%3AInternal+is%3Aclosed, I don't see anything I really want listed on a release note.  Nice as a developer to have something that says "don't put this in the release notes" just as it's nice to have `highlight` to say "this one should be called out in the release note"

File: release-checks.sh [hunk 2]
```diff
@@ -52,6 +52,12 @@ then
     exit 1
```
Comment (danielmitterdorfer) [inline release-checks.sh:52]: In the line above, we still mention `RELEASE.md` but it is gone by now. Can you please remove it as well?

File: changelog.py [hunk 3]
```diff
@@ -105,9 +105,10 @@ def main():
...
+    print_category("Bug Fixes", prs(gh, milestone, with_labels="bug", without_labels=[":Docs", "highlight", ":internal"]))
```
Comment (danielmitterdorfer) [inline changelog.py:109]: The `without_labels` list needs to include the new label `:misc` as well.

File: changelog.py [hunk 4]
```diff
@@ -105,9 +105,10 @@ def main():
...
     print_category("Doc Changes", prs(gh, milestone, with_labels=":Docs", without_labels=["highlight"]))
```
Comment (danielmitterdorfer) [inline changelog.py:110]: The `without_labels` list needs to include the new label `:misc` as well.

File: changelog.py (hunk 1 above)
[the remark this answers is not in the document]
Comment (danielmitterdorfer) [reply changelog.py:108]: The `without_labels` list needs to include the new label `:misc` as well.

**Review by danielmitterdorfer — COMMENTED** [review]
Thanks! I left a few comments.

**Review by danielmitterdorfer — APPROVED** [review]
Thanks! LGTM

**Pull request merged**